src/sql/magic_cmd.py

```diff
@@ -51,6 +52,8 @@ def _validate_execute_inputs(self, line):
             "explore",
             "snippets",
         ]
+        CONNECTION_INDEPENDENT_COMMANDS = ["snippets"]
+        SUPPORTED_BY_BOTH_COMMANDS = ["profile"]
```

Comment:
what does supported by both commands mean? I guess you meant "supported by sqlalchemy and dbapi connections"?

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/804/files#diff-2aa72d6ecafc2697ec8174353868dae816494131c70b7e48be6115fda9f61370R62)

Changed to `COMMANDS_SQLALCHEMY_ONLY`